content/blog/2020-09-22-treedata.table/index.md

```diff
@@ -0,0 +1,82 @@
+Data manipulation in treedata.table is significantly faster than in other commonly used packages such as base (>35%), treeplyr (>60%), and dplyr (>90%).
+Additionally, treedata.table is >400% faster than treeplyr during the initial data/tree matching step (Fig. 1B).
+
+
+<div style="text-align:center">
+<img src='img/bench_TDT_Aug14.png' alt="Six panel figure showing comparisons of benchmark data for different phylogenetic data processing workflows" align="middle"width="600" />
+</div>
+
+ <font size="2"> **Fig. 1.** Results for the treedata.table microbenchmark during (**A**) data manipulation (treedata.table[) and (**B**) tree/data matching steps.
+  We compare the performance of treedata.table against data.table, base, treeplyr, and dplyr using the microbenchmark R package.</font>
+
+
+### What can I do in treedata.table?
+
+The simplest function that can be done in treedata.table is matching the tips of a phylogenetic tree to
+the rows in a data.table object.
+Like many R packages involving phylogenetic trees, treedata.table assumes that your tree is an object of class
+phylo.
+The first vignette (in English [here](https://ropensci.github.io/treedata.table/articles/AA_treedata.table_intro_english.html) and Spanish [here](https://ropensci.github.io/treedata.table/articles/AB_treedata.table_intro_spanish.html)) walks new users through matching a phylogeny to a data.table, and performing some common data operations, such as selecting data columns and dropping tips that do not have data.
+There are other ways to achieve similar results, but as seen on Fig. 1, these are not as efficient.
+
+Something unique in treedata.table is the ability to work with multiPhylo objects.
+multiPhylo objects, as their name suggests, are colelctions of trees.
+For example, perhaps you would like to match all the trees in a Bayesian posterior sample to a dataset to perform
+a comparative methods analysis for every tree in your sample.
+Our second [vignette](https://ropensci.github.io/treedata.table/articles/B_multiphylo_treedata.table.html) walks researchers through the process of syncing many trees to one dataset, manipulating the data, and performing a continuous trait evolution modeling exercise in [geiger](https://cran.r-project.org/web/packages/geiger/index.html).   
+
+Our other two vignettes cover important data functions researchers may wish to do with phylogenetic data.
+For example, syncing names across datasets and phylogenetic trees based on [partial matching](https://ropensci.github.io/treedata.table/articles/C_PartialMatching.html), and [assorted data tidying functions](https://ropensci.github.io/treedata.table/articles/D_AdditionalFunctions_treedata.table.html).
+
+### Where did treedata.table come from?
+<div style="text-align:center">
+<img src='img/NantucketBeach.jpg' alt="Beach with small waves" align="middle"width="600" />
+</div>
+<font size="2"> **Fig. 2.** The beach outside the workshop bunkhouse. </font>
```

Comment:
Figured it out! for this to render properly there needs to be  an empty line between the </div> and the <font size...>
```suggestion

<font size="2"> **Fig. 2.** The beach outside the workshop bunkhouse. </font>
```